src/index.html

```diff
@@ -13,6 +13,51 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="display">
```

Comment:
Use 2 spaces for indentation in your file. The `div` element on line 16 should be indented with 2 spaces as it is a child of the `body` element.